src/wrap-page.js

```diff
@@ -1,8 +1,16 @@
 const React = require('react');
 const { Helmet } = require('react-helmet');
 
+const isExcluded = (collection, element) =>
+  Array.isArray(collection) &&
+  collection.indexOf(element.replace(/\/+$/, '')) !== -1;
```

Comment:
Please use [`.includes`](https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Array/includes) instead of `.indexOf(...) !== -1`.

Wouldn't it be better to allow passing regex instead of plain strings?

Reply:
> Please use .includes instead of .indexOf(...) !== -1.

Sure !

```suggestion
  collection.includes(element.replace(/\/+$/, ''));
```
---

> Wouldn't it be better to allow passing regex instead of plain strings?

Yes it would. But it may be a second step :)